README.adoc

```diff
@@ -157,46 +150,6 @@ OVN/OVS is an optional dependency. If you want to use it, check the requirements
 ovirt-engine.spec.in file for a list of packages. Otherwise, you should reply 'No'
 when asked about it by engine-setup.
 
-For Fedora prerequisites installation, following commands can be applied:
```

Comment:
There is also Fedora mentioning in a `Initialize PostgreSQL` section, I will soon check these instruction today or tomorrow while installing engine from scratch on CenOS Stream 8 and update if needed. As far as I remember there was some slightly different syntax for the `sudo postgresql-setup --initdb --unit postgresql` command, but I'm not sure.

Reply:
You can just remove the `# fedora` comment under the `PostgreSQL accessibility` or resolve this comment and I will handle it in the follow-up patch :)

Reply:
I have removed the comment, you can update the syntax in a follow-up patch, but at least mentions to Fedora are all cleaned-up with this PR.

Reply:
Thanks, the syntax is correct :)